misc/measure.py

```diff
@@ -26,15 +26,17 @@
 devnull = open('/dev/null', 'w')
 
 def run(cmd, repeat=10):
-    print('sampling:', sys.stdout.flush())
+    print('sampling:', end='')
+    sys.stdout.flush()
 
     samples = []
     for _ in range(repeat):
         start = time.time()
         subprocess.call(cmd, stdout=devnull, stderr=devnull)
         end = time.time()
         dt = (end - start) * 1000
-        print('%dms' % int(dt), sys.stdout.flush())
+        print('%dms' % int(dt))
```

Comment:
Don't want to be nit-picking, and I haven't used `measure.py` myself so I don't know how important it is. But, I think you're missing the `end=' '` argument in this line still. :) The old code looked like this:

```python
        print '%dms' % int(dt),
        sys.stdout.flush()
```

PS: And as I mentioned in a comment for this merge request, I did a mistake myself there and used `end=''` instead of the correct `end=' '` (note that string with a single whitespace character).